install_prerequisites/build

```diff
@@ -166,9 +201,13 @@ set_url()
 {
   if [[ $package_to_build == 'cmake' ]]; then
     major_minor="${version_to_build%.*}"
+  elif [[ "$package_to_build" == "gcc" && "$version_to_build" == "$default_version" ]]; then
+    gcc_url_head="ftp.lip6.fr:pub/gcc/releases/gcc-5.3.0/"
```

Comment:
Would you mind changing this to http://ftpmirror.gnu.org/ ? (Unless of course you think it's a bad idea...)

From http://www.gnu.org/prep/ftp.en.html :
> You can use the generic url http://ftpmirror.gnu.org to automatically choose a nearby and up-to-date mirror.

You can either author a new commit, or amend the latest commit via:

```shell
$ # update url in install_prerequisites/build
$ git add install_prerequisites/build
$ git commit --amend
$ git push --force origin
```

`git commit --amend` will change the most recent commit to reflect any additional changes you `git add`ed and allow you to edit the commit message.